core/frontend/src/tile/map/MapLayerImageryProvider.ts

```diff
@@ -33,32 +32,46 @@ export abstract class MapLayerImageryProvider {
   protected _hasSuccessfullyFetchedTile = false;
   public status: MapLayerImageryProviderStatus = MapLayerImageryProviderStatus.Valid;
   public readonly onStatusChanged = new BeEvent<(provider: MapLayerImageryProvider) => void>();
+
+  /** @internal */
   private readonly _mercatorTilingScheme = new WebMercatorTilingScheme();
+  /** @internal */
   private readonly _geographicTilingScheme = new GeographicTilingScheme();
 
   public get tileSize(): number { return this._usesCachedTiles ? tileImageSize : untiledImageSize; }
   public get maximumScreenSize() { return 2 * this.tileSize; }
-  public get minimumZoomLevel(): number { return 0; }
-  public get maximumZoomLevel(): number { return 22; }
+  public get minimumZoomLevel(): number { return this.defaultMinimumZoomLevel; }
+  public get maximumZoomLevel(): number { return this.defaultMaximumZoomLevel; }
   public get usesCachedTiles() { return this._usesCachedTiles; }
   public get mutualExclusiveSubLayer(): boolean { return false; }
   public get useGeographicTilingScheme() { return false;}
   public cartoRange?: MapCartoRectangle;
+
+  // Those values are used internally for various computation,
```

Comment:
Should they be marked `@internal` then?

Reply:
yes for sure.  That will get revisited in an upcoming PR anyway.